stdlib.sh

```diff
@@ -104,6 +110,14 @@ EOC
 #
 # ... near the top of the calling script.
 
+declare std_PRETEND
+# Standard usage is:
+#
+std_PRETEND="${PRETEND:-0}"
+
+declare std_WRAPLOG
+std_WRAPLOG="${WRAPLOG:-0}"
```

Comment:
The default prior to this change _was_ to wrap, so the default should reflect this.  I'm also thinking that `std_WRAPOUTPUT` might be a better name?